### ISSUE TITLE
OUT OF DATE: Selenium deprecated find_elements_by_

### ISSUE THREAD

**Issue by robbiedobbie1** [jimdevops19/SeleniumSeries#2]
- Selenium deprecated **find_elements_by_** and instead implements **find_elements_**  

-ALSO: The test URL's  supplied to selenium website offer "Page not found -404"
EG: **Home >> Test >> Page not found
The requested page "/test/jquery-download-progress-bar-demo.html**" could not be found.

**Comment by Rocking80** [comment]
Yes, selenium use `find_element` now. Need to import `from selenium.webdriver.common.by import By`
